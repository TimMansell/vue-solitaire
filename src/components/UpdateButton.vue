<template>
  <Button @click="updateApp()" data-test="update-btn"> Update </Button>
</template>

<script>
import Button from './Button.vue';

export default {
  name: 'UpdateButton',
  components: {
    Button,
  },
  methods: {
    updateApp() {
      window.location.href = '/';
    },
  },
};
</script>
