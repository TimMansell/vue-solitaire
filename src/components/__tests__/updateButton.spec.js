import { shallowMount } from '@vue/test-utils';
import UpdateButton from '@/components/UpdateButton.vue';

describe('UpdateButton.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(UpdateButton);

    expect(wrapper).toMatchSnapshot();
  });
});
