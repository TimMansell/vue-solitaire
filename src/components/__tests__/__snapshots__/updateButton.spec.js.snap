// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UpdateButton.vue matches snapshot 1`] = `<button-stub type="default" size="md" data-test="update-btn"> Update </button-stub>`;
